presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainOrcPredicate.java

```diff
@@ -185,8 +187,14 @@ public boolean matches(long numberOfRows, Map<Integer, ColumnStatistics> statist
             }
         }
 
+        // did we read any bloom filter?
+        if (numberOfBloomFiltersChecked < 1) {
```

Comment:
Isn't it covered already by https://github.com/RobinUS2/presto/blob/83cbb3376875f41e68ae224db269024adeccbf4e/presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainOrcPredicate.java#L156?
And maybe when all column references have `domain = all`?

I'd prefer explicit check based on the actual conditions (e.g. why those filters where not read: either there are no bloom filters OR predicate domains allow all values) instead of this counter.